runtimes/peregrine/src/lib.rs

```diff
@@ -689,53 +692,53 @@ construct_runtime! {
 		UncheckedExtrinsic = UncheckedExtrinsic,
 	{
 		// Basic stuff; balances is uncallable initially.
-		System: frame_system::{Pallet, Call, Config, Storage, Event<T>} = 0,
-		RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage} = 1,
+		System: frame_system = 0,
```

Comment:
Not sure about this change. Maybe we rather revert it. There could be an unintentional change hidden here. E.g. we didn't include `call` for a specific pallet but now it's included.

Reply:
To be honest, I don't see the advantage of having them. If we always use all components, it's only error prone to list them manually. if there are cases where we don't what specific parts of the pallet, it's not clear if we left them out by mistake or which pallets are not included completely.

Reply:
I would say that we could just reproduce what we have in the current runtime in the new syntax? Meaning that we could manually remove the elements that we have not manually added to the current configuration, since now components are included by default. We will have to make the switch sooner or later anyway.

Reply:
For instance `cargo expand`ing both runtimes to see if there's anything different by just using the default values.

Reply:
> For instance `cargo expand`ing both runtimes to see if there's anything different by just using the default values.

that'll be interesting! Actually I don't expect to see a difference. I don't know if we excluded something. 🙈

Reply:
So the files are really messy! But the difference is:

* Call: ParachainInfo & AuraExt are added. They both don't have any calls to call.
* StorageMetadata: Aura, AuraExt, Utility & Inflation are added
* Genesis Config: Indices, TransactionPayment, Scheduler, are added
* Inherent check: Authorship
* ValidateUnsigned: ParachainSystem

Reply:
So it is not worth to exclude them explicitly to maintain the same runtime?

Reply:
You can compare the runtimes here: All is with all parts of a pallet specified. Selected is with only the parts that differ specified. There are still a few changes. I think the StorageMetadata is actually a good addition, but the ParachainSystem call is a bit strange. 🤔 

[Archive.zip](https://github.com/KILTprotocol/mashnet-node/files/7879819/Archive.zip)

Reply:
> So the files are really messy! But the difference is:
> 
> * Call: ParachainInfo & AuraExt are added. They both don't have any calls to call.
> * StorageMetadata: Aura, AuraExt, Utility & Inflation are added
> * Genesis Config: Indices, TransactionPayment, Scheduler, are added
> * Inherent check: Authorship
> * ValidateUnsigned: ParachainSystem

Really interesting! I would propose to either the differences explicitly or with the new implicit way, except for the calls.